script/Propose.s.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
```

Comment:
*(arbitrary comment)*

- This is the same as gitcoin except I updated the addresses and proposer comment.